_config/config.yml

```diff
@@ -41,6 +41,24 @@ FileField:
   extensions:
     - MultisitesFileFieldExtension
 ---
+Name: sisspamprotectionconfig
+After: 'framework/*','cms/*'
+---
+FormSpamProtectionExtension:
+  default_spam_protector: '\StudioBonito\SilverStripe\SpamProtection\Honeypot\SpamProtector\HoneypotSpamProtector'
+UserDefinedForm:
+  extensions:
+    - BasisUserDefinedFormSpamProtectionExtension
+UserForm:
+  extensions:
+    - BasisUserFormSpamProtectionExtension
+Form:
+  extensions:
+    - BasisFormProtectionExtension
+BasisFormProtectionExtension:
```

Comment:
Yeah I seem to recall that discussion. @Neumes, can the extension application be changed to just a user defined form ?